lib/pghero/methods/settings.rb

```diff
@@ -32,6 +50,10 @@ def vacuum_settings
       def fetch_settings(names)
         Hash[names.map { |name| [name, select_one("SHOW #{name}")] }]
       end
+
+      def citus_fetch_worker_settings(citus_worker_names)
+        Hash[citus_worker_names.map { |citus_worker_name| [citus_worker_name, select_one("SELECT result FROM run_command_on_workers($cmd$ SHOW #{citus_worker_name} $cmd$) LIMIT 1")] }]
```

Comment:
These are setting names, not worker names, right?

Reply:
`fetch_settings` uses `names` as an argument so the added method for workers uses the same "names" for `citus_workers`. Maybe I can just leave it as `names` in the new method as well because it is misleading as you pointed out.